### ISSUE TITLE
Possibly separate template editor into separate components

### ISSUE THREAD

**Issue by martinjoconnor** [metadatacenter/cedar-template-editor#599]
Think about separating into template creator, editor, workbench etc.

**Comment by willrett** [comment]
Yes, the single application should be split into three micro front-ends.
   
https://medium.com/@tomsoderlund/micro-frontends-a-microservice-approach-to-front-end-web-development-f325ebdadc16

**Comment by martinjoconnor** [comment]
Work ongoing on Template Editor. May start on Workspace soon thereafter.
